src/ir/context.rs

```diff
@@ -577,6 +577,14 @@ If you encounter an error missing from this list, please file an issue or a PR!"
         }
     }
 
+    /// Returns `true` if the target architecture is wasm32
+    pub fn is_target_wasm32(&self) -> bool {
```

Comment:
This won't work if building with clang 4 or 3.9 (target_info will be null), but maybe that's fine.

Reply:
I doubt that wasm backend even existed for those versions of clang and as such I don't think what would be an issue.

Reply:
We still need to update the test expectations for these two versions, as we run CI on those. To do that you need to add the file in `tests/expectations/tests/libclang-3.9` (and same for 4).

With that bit, this patch looks great, thank you!